src/pages/index.js

```diff
@@ -30,8 +43,13 @@ const newCardTitle = document.querySelector(".modal__input-title");
 const newCardURL = document.querySelector(".modal__input-url");
 const profileForm = document.querySelector("#edit-profile-form");
```

Comment:
**Could be improved**

We can get any form using their **unique** attribute `name` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms